central/scannerdefinitions/handler/handler.go

```diff
@@ -140,6 +136,24 @@ func init() {
 	var err error
 	scannerUpdateBaseURL, err = url.Parse("https://definitions.stackrox.io")
 	utils.CrashOnError(err) // This is very unexpected.
+
+	// Parse the main version number variants, continue in a broken state if version
+	// is not parseable.
```

Comment:
This is only for release builds, right? For dev it looks like we crash (`utils.ShouldErr`)? Also, for release, it'd be VERY surprising if this failed, so I guess this is ok

Reply:
Yes, for release we continue, for a "just in case" scenario since this would only affect offline bundle uploading.